pkg/dazzle/combiner.go

```diff
@@ -148,10 +148,8 @@ func (p *Project) Combine(ctx context.Context, chunks []string, dest reference.N
 
 	now := time.Now()
 	ccfg := ociv1.Image{
-		Created:      &now,
-		Architecture: basecfg.Architecture,
-		History:      allHist,
-		OS:           basecfg.OS,
```

Comment:
Removing `Architecture` and `OS` seems odd, I'm not sure if this will be breaking, it'll be interesting to test :soon:

Reply:
As a next step, I'd like to get this merged, so we can release, and test in `workspace-images`, and then test those images in `gitpod`.

Reply:
:brain: gonna double check, to see if these fields were moved, and can be exposed elsewhere

Reply:
Flipping back to draft, failing unit tests

Reply:
These fields are stored in the image descriptor's platform field, which we push as part of the manifest, after pushing the blob.

The definition for the platform field: https://pkg.go.dev/github.com/opencontainers/image-spec@v1.1.0-rc3/specs-go/v1#Platform

Where we set the Platform in the descriptor for the manifest:
https://github.com/gitpod-io/dazzle/blob/a2b54c8e100f661239b01c4978d20249cebf61af/pkg/dazzle/combiner.go#L194